package.json

```diff
@@ -77,5 +77,10 @@
 		"playwright": "^1.42.1",
 		"react-modal": "^3.16.1",
 		"react-tooltip": "^5.26.3"
+	},
+	"dependencies": {
+		"axios": "^1.7.2",
+		"leaflet": "^1.9.4",
+		"react-leaflet": "^4.2.1"
```

Comment:
Let's move these to devDependencies like rest.

Reply:
@mauteri Fixed up the package.json file

Reply:
resolved! marking conversation as resolved at this time.